x-pack/plugins/security_solution/public/detection_engine/rule_management/logic/types.ts

```diff
@@ -202,3 +203,8 @@ export interface FindRulesReferencedByExceptionsProps {
   lists: FindRulesReferencedByExceptionsListProp[];
   signal?: AbortSignal;
 }
+
+export enum RuleSourceTypesEnum {
```

Comment:
We already have a rules source type that can be either `internal` or `external`. To avoid confusion with this enum, let's rename it to something like `RuleCustomizationEnum`.